download-NC.py

```diff
@@ -80,36 +80,18 @@
 INTO TABLE {config['NC']['table']}
 CHARACTER SET latin1
 FIELDS TERMINATED BY ','
+ENCLOSED BY '"'
```

Comment:
Should this also be `OPTIONALLY ENCLOSED`?

Reply:
Maybe? I didn't really do this part so not sure what optionally enclosed does but probably doesn't hurt to include it...there were some errors in the csv where it wasn't enclosed by a "" for some of them right?

Reply:
I didn't put the optionally since it seemed like every field was enclosed by "", but we can change it

Reply:
We can keep it for now.